cpp/include/cudf/detail/hashing.hpp

```diff
@@ -44,6 +44,12 @@ std::unique_ptr<column> murmur_hash3_32(
   rmm::cuda_stream_view stream        = cudf::default_stream_value,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+std::unique_ptr<column> spark_murmur_hash3_32(
```

Comment:
I wonder why the APIs here don't have doxygen?

Reply:
They're detail APIs, which don't require docs.

Reply:
The _build_ does not require detail APIs to have doxygen but programmers would still appreciate documentation.
You can see many detail functions are documented with `@copydoc` tags for example.
I can add some _detail_ to the https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/docs/DOCUMENTATION.md

Reply:
There will be some significant changes in this file with other planned work (#11296), so I'm going to defer on this until I can do it for the whole file. I added a note to myself to improve this later for all hash functions. https://github.com/rapidsai/cudf/issues/10081#issuecomment-1197425159